core/src/components/range/range.ionic.scss

```
@@ -107,12 +107,11 @@
 }
 
 .range-pin {
-  @include transform(translate3d(0, 100%, 0), scale(0.01));
-  @include padding(globals.$ion-space-200, globals.$ion-space-200, globals.$ion-space-200, globals.$ion-space-200);
+  @include padding(globals.$ion-space-300, globals.$ion-space-300, globals.$ion-space-300, globals.$ion-space-300);
```

```suggestion
  @include padding(null, null, globals.$ion-space-100, null);
```

We only need the bottom padding to separate the knob from the text